roles/update/defaults/main.yml

```diff
@@ -17,3 +17,8 @@
 
 # All variables intended for modification should be placed in this file.
 # All variables within this role should have a prefix of "cifmw_update"
+
+cifmw_update_openstack_update_run_target_version: "0.0.2"
+cifmw_update_openstack_update_run_operators_updated: true
+cifmw_update_openstack_update_run_containers_namespace: "podified-antelope-centos9"
+cifmw_update_openstack_update_run_containers_target_tag: "current-podified"
```

Comment:
update_openstack_update

is that really necessary?

but this is totally subjective and this is just me thinking out loud not blocking on any naming preference here...